wercker.yml

```diff
@@ -25,6 +25,9 @@ generate:
   steps:
     - wercker/setup-go-workspace:
         package-dir: github.com/pusher/faros
+    - script:
+        name: configure make
+        code: ./configure
```

Comment:
Should we not be using the new `prepare-env-XYZ` here (as it currently fails because `dep` isn't installed)?